pkg/apis/dynatrace/v1alpha1/oneagentapm_types.go

```diff
@@ -24,6 +24,20 @@ import (
 // +k8s:openapi-gen=true
 type OneAgentAPMSpec struct {
 	BaseOneAgentSpec `json:",inline"`
+
+	// Optional: Custom OneAgent docker image
```

Comment:
Would add Code Modules on the description, since it's not the same as the full-stack OneAgent image